kratos/processes/assign_scalar_variable_to_entities_process.h

```diff
@@ -156,8 +156,6 @@ class KRATOS_API(KRATOS_CORE) AssignScalarVariableToEntitiesProcess
     ///@{
 
     /// Copy constructor.
-    AssignScalarVariableToEntitiesProcess(AssignScalarVariableToEntitiesProcess const& rOther);
```

Comment:
This is defined in the cpp?

Reply:
I don't think so
I guess the template is wrong

Reply:
Nope, apparently nope

Reply:
I could not find it

Reply:
the c++11 way would anyway be to delete them and not to put them in private